### ISSUE TITLE
Implement a nice way to show the user that there was an error during the sourcing of `init.vim`

### ISSUE THREAD

**Issue by qvacua** [qvacua/vimr#251]
With GH-245 VimR does not crash when there was an error with `init.vim`. However, VimR just ignores the error and does not show the user that there was an error. Think about a nice way to do this.

Probably somewhere near `case NeoVimServerMsgIdNeoVimReady:` in `NeoVimAgent.m`.

**Comment by qvacua** [comment]
Just show an alert... (not nice...)

**Comment by qvacua** [comment]
https://github.com/qvacua/vimr/releases/tag/snapshot%2F20160825.2258-64

**Comment by romaninsh** [comment]
Wasn't fixed for me in build 64 (compiled from master)

![error-not-shown](https://cloud.githubusercontent.com/assets/453929/18000168/ac29c8d0-6b74-11e6-8ca7-dda18af1456c.gif)

**Comment by romaninsh** [comment]
![compile error](https://cloud.githubusercontent.com/assets/453929/18000198/d8836e22-6b74-11e6-8e36-5b65993d1ffc.png)

Here is inspiration for error message.

**Comment by qvacua** [comment]
1. I'll look into that: is a ctrlp-setting causing the error? Can you post some more details such that I can try it myself?
2. Yes, I was just too lazy... 😳 I plan to use the OSX's native user notification facilities 😀.

**Comment by romaninsh** [comment]
I noticed that it's only when you run through 'vimr' that you don't get error.

**Comment by romaninsh** [comment]
ctrp is included from init. I can also reproduce it if i put some garbage into "init"

**Comment by romaninsh** [comment]
setting a high bar for the UI from the very start is a good way to have a best neovim UI for mac!

**Comment by kenny-evitt** [comment]
Are there any workarounds for viewing the error messages at all?

**Comment by qvacua** [comment]
You can use `:messages` to view the errors caused by `init.vim`.

**Comment by kenny-evitt** [comment]
@qvacua Yes, thanks. After installing the latest version of VimR, it shows a nice popup message with that same suggestion too.
